examples/alerts/alerts.md

```diff
@@ -0,0 +1,143 @@
+# Alerts
+
+Here are some example alerts configured for Kubernetes environment.
+
+## Compaction
+
+```
+- alert: ThanosCompactHalted
+  expr: thanos_compactor_halted{app="thanos-compact"} == 1
+  for: 5m
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos compaction has failed to run and now is halted
+    impact: Long term storage queries will be slower 
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: COMPACTION_URL
+- alert: ThanosCompactCompactionsFailed
+  expr: rate(prometheus_tsdb_compactions_failed_total{app="thanos-compact"}[5m]) > 0
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Compact is failing compaction
+    impact: Long term storage queries will be slower 
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: COMPACTION_URL
+- alert: ThanosCompactBucketOperationsFailed
+  expr: rate(thanos_objstore_bucket_operation_failures_total{app="thanos-compact"}[5m]) > 0
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Compact bucket operations are failing
+    impact: Long term storage queries will be slower 
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: COMPACTION_URL
```

Comment:
what about no compaction being made in very long time?

Reply:
plus up{compactor} == 0 + absent

Reply:
plus "more than one compactor is running for the same bucket (critical)" - wonder if we have metrics for that